tests/slate/test_variational_prb.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import numpy as np
+from firedrake import *
+
+
+@pytest.mark.parametrize('degree', range(1, 4))
```

Comment:
Maybe only `[1, 2]`? You don't expect the degree to make a difference to whether or not this bug exhibits, right?